### ISSUE TITLE
Make a button in Global Config to Clean PHPThumb Cache

### ISSUE THREAD

**Issue by Igcorreia** [FLEXIcontent/flexicontent-cck#217]
Usually when moving a site the PHPThumb cache caches the URL of the images with the OLD Path and sometime is takes 1 hour to delete the cache via FTP. 

Would be nice to have a button that clean PHPThumb cache via FC for people without access to the SSH or Cpanel.

**Comment by ggppdk** [comment]
You mean a button in FLEXIcontent dashboard

**Comment by Igcorreia** [comment]
No I really mean in the Global Configuration, I don't want the user/client touching this setting. 

We only need this for 2 reasons:
- change in the field values and we need to redesign all images 
- Moving the site from dev to production - PhpThumb caches the FULLPATH so it break the site when it goes live.

**Comment by ggppdk** [comment]
Ok, it will be a button in Dashboard that will be displayed to those having permission to configure the component  ))

**Comment by micker** [comment]
i think the best place is in field not ? clean only image create by the field

**Comment by ggppdk** [comment]
yes but the phpThumb cache is not used by fields for frontend display, so it is a different thing,

- phpTumb is used in item form for thumbnails (image field)
- fileselement view, filemanager view
- frontend templating, e.g. blog template
- universal module templates

**Comment by Igcorreia** [comment]
:( if I have 20 fields I will need then to open all of them and clean the cache from each.

What I mean is CACHE CLEANER.
What Micker mean is THUMBNAIL RE-GENERATOR.

Is the same result but our objectives are different. Won't you agreed?
